spec/cypress/support/accessible-paths.json

```diff
@@ -0,0 +1,37 @@
+[
```

Comment:
Can we retrieve a list of paths from Rails at test runtime? 

https://guides.rubyonrails.org/routing.html#inspecting-and-testing-routes has some interesting ideas

I really don't want to re-create our routes file in json...

Reply:
We could maybe replace it with that some day, but at the moment, not all routes are tested/accessible.